nemo_text_processing/text_normalization/rw/verbalizers/time.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+# Copyright (c) 2024, DIGITAL UMUGANDA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pynini
+from pynini.lib import pynutil
+from nemo_text_processing.text_normalization.en.graph_utils import (
+    GraphFst,
+    delete_space,
+    NEMO_CHAR
+)
+
+class VerbalizeTimeFst(GraphFst):
+    def __init__(self):
+        super().__init__(name="time",kind="verbalize")
+        hour = (pynutil.delete("hours:")+delete_space+pynutil.delete("\"")+pynini.closure(NEMO_CHAR,1,60)+pynutil.delete("\"")+delete_space \
```

Comment:
You can just do base closure since the `hours:` property will limit potential tokens. I also believe it is more efficient since counting in FST graphs enlarges the graph (iirc).